defi/src/api2/index.ts

```diff
@@ -33,6 +34,10 @@ async function main() {
     .then(() => {
       console.timeEnd('Api Server init')
       console.log('Webserver started on port ' + port)
+      try {
+        const currentCommitHash = fs.readFileSync(__dirname + '/../../.current_commit_hash', 'utf8')
+        console.log('current code hash: ', currentCommitHash)
```

Comment:
isnt it better to display the commit being used?

Reply:
This will be the commit being used, when we rollback to safe commit, current commit will match that